ui/src/jobs/containers/DetailPage.jsx

```diff
@@ -0,0 +1,131 @@
+import InlineUL from '../../common/components/InlineList/InlineUL';
+
+class DetailPage extends Component {
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(fetchJob(this.recordId));
+  }
+
+  componentDidUpdate(prevProps) {
+    const { dispatch } = this.props;
+    const prevRecordId = prevProps.match.params.id;
+    if (this.recordId !== prevRecordId) {
+      dispatch(fetchJob(this.recordId));
+    }
+  }
+
+  get recordId() {
+    const { match } = this.props;
+    return match.params.id;
+  }
+
+  render() {
+    const { record, loading } = this.props;
+    const metadata = record.get('metadata');
+    const created = record.get('created');
+    const updated = record.get('updated');
+    if (!metadata) {
+      return null;
+    }
+    const position = metadata.get('position');
+    const institutions = metadata.get('institutions');
+    const regions = metadata.get('regions');
+    const arxivCategories = metadata.get('arxiv_categories');
+    const ranks = metadata.get('ranks');
+    const experiments = metadata.get('accelerator_experiments');
+    const deadlineDate = metadata.get('deadline_date');
+    const description = metadata.get('description');
+    const status = metadata.get('status');
+    return (
+      <Fragment>
+        <Row type="flex" justify="center">
+          <Col className="mv3" xs={24} md={21} lg={19} xl={18}>
+            <ContentBox
+              loading={loading}
+              rightActions={
+                <Fragment>
+                  <div className="pr2">
+                    <CreatedDate created={created} />,{' '}
+                    <UpdatedDate updated={updated} />
+                  </div>
+                </Fragment>
+              }
+            >
+              {status === 'closed' && (
+                <Alert
+                  type="error"
+                  message={<span>This job is closed!</span>}
+                  showIcon={false}
+                />
+              )}
+              <h2 className="mt3 b">{position}</h2>
+              <div className="mt1">
+                <InlineUL separateItemsClassName="separate-items-with-middledot">
+                  {institutions && (
+                    <InstitutionsList institutions={institutions} />
+                  )}
+                  <RegionsList regions={regions} />
+                </InlineUL>
+              </div>
+              <div className="mt2">
+                <ArxivCategoryList
+                  arxivCategories={arxivCategories}
+                  wrapperClassName="di"
+                />
+                <InlineUL
+                  separateItemsClassName="separate-items-with-middledot"
+                  wrapperClassName="di"
+                >
+                  {ranks && <RanksList ranks={ranks} />}
+                  {experiments && <ExperimentList experiments={experiments} />}
+                </InlineUL>
+              </div>
+              <div className="mt2 b">
+                <IconText
```

Comment:
this needs a component of its own, since it's like to that `deadline` will need formatting.